TehanovAnton/hometask-2/get_middle_charachter.rb

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+ERROR_MESSAGE = "String can't be blank"
+ENTER_PROMPT = 'Enter string:'
+EXIT = 'exit!'
+
+def get_middle(string)
+  half = string.size / 2
+  string.length.odd? ? string[half] : string[(half - 1)..half]
+end
+
+def error_message
+  ERROR_MESSAGE
+end
+
+def result(input)
+  "Middle character: #{get_middle(input)}"
+end
+
+def run_cli
```

Comment:
@TehanovAnton you already should have a concept of a loop through CodeAcademy lessons. Can you try to rewrite it with that context in mind?